app/models/post.rb

```diff
@@ -1,6 +1,7 @@
 class Post < ApplicationRecord
 
     belongs_to :user
+    belongs_to :category,       optional: true
```

Comment:
dlaczego optional true?

chyba wymagamy zeby kazdy post nalezal do kategorii?

jesli masz problem z tym jak zrobic migracje ze starymi postami, to mozesz je z automatu dodac do category np "old_main" albo jakiegos innego nieuzywanego potem subreddita.